tests/test_testing.py

```diff
@@ -183,3 +184,21 @@ def cli_no_error():
     result = runner.invoke(cli_no_error)
     assert result.exit_code == 0
     assert result.output == 'hello world\n'
+
+
+def test_env():
+    @click.command()
+    def cli_env():
+        click.echo('ENV={}'.format(os.environ['TEST_CLICK_ENV']))
```

Comment:
We have to run under Python 2.6, so please change this to `"ENV=%s" % os.environ['TEST_CLICK_ENV']`